### PR TITLE
[CBRD-24896] Problem when accesing system table/views with search condition comparing set values

### PR DESCRIPTION
**Pull request by shparkcubrid**
http://jira.cubrid.org/browse/CBRD-24896

- Modified to determine that index scan is impossible in the following cases
```
set = set
{col1,col2} = {1,2}
```
- fix so that only collection types written in parentheses can index-scan.

### CODE REVIEW

File: src/optimizer/query_graph.c [hunk 1]
```diff
@@ -2195,20 +2195,14 @@ qo_analyze_term (QO_TERM * term, int term_type)
 		  lhs_indexable = true;
 		}
 	    }
-	  else if (pt_is_multi_col_term (lhs_expr))
+	  else if (pt_is_multi_col_term (lhs_expr) && lhs_expr->flag.is_paren)
 	    {
 	      /* multi column case (attr,attr,...) is indexable for RANGE, EQ operation */
```
Comment (youngjinj) [inline src/optimizer/query_graph.c]: It is trivial.
The comment about 'EQ operation' should be removed.

Comment (shparkcubrid) [inline src/optimizer/query_graph.c]:
```suggestion
	      /* multi column case (attr,attr,...) is indexable for RANGE operation */
```

Comment (shparkcubrid) [reply src/optimizer/query_graph.c]: I fixed it.

File: src/optimizer/query_graph.c [hunk 2]
```diff
@@ -2249,7 +2243,7 @@ qo_analyze_term (QO_TERM * term, int term_type)
 			}
 		      segs++;
 		    }
-		  if (!is_find_local_name)
+		  if (!is_find_local_name || segs <= 1)
```
Comment (youngjinj) [inline src/optimizer/query_graph.c:2246]: Are there test scenarios that require this change?

**Review by youngjinj — APPROVED** [review]

**Review by xmilex-git — APPROVED** [review]

**Pull request merged**